spec/support/shared_context/rake_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+# https://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+require "rake"
+
+shared_context "rake" do
```

Comment:
You should exclude that file (or it's directory) from SimpleCov so it's not included in coverage stats.